package.json

```diff
@@ -27,8 +27,9 @@
     "cucumber": "./node_modules/.bin/cucumber-js --publish ./cucumber/features/*.feature -f json:./reports/cucumber_report.json && node ./cucumber/index.js",
     "cucumber:ci": "cross-env TTA_SMART_HUB_URI=http://localhost:3000 CUCUMBER_USER=$CUCUMBER_USER CUCUMBER_USER_ID=999999 yarn cucumber",
     "db:migrate": "node_modules/.bin/sequelize db:migrate",
-    "db:migrate:ci": "cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub node_modules/.bin/sequelize db:migrate",
+    "db:migrate:ci": "cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub node_modules/.bin/sequelize db:migrate && yarn db:seed:all",
```

Comment:
So I've been confused about this for awhile now. `cross-env` is only used to supply command line vars in windows systems right? None of the CI environments are windows, we use linux boxes, so we shouldn't need this. Also, we are supplying the `POSTGRES_USERNAME` and `POSTGRES_DB` vars via `docker-compose-test` for the `dynamic_security_scan` job, so we shouldn't need them as command line vars here.